ignite/cmd/network_chain_publish.go

```diff
@@ -74,6 +73,11 @@ func networkChainPublishHandler(cmd *cobra.Command, args []string) error {
 		rewardDuration, _         = cmd.Flags().GetInt64(flagRewardHeight)
 	)
 
+	source, err := xurl.HTTPS(args[0])
```

Comment:
Can we also implement an `xurl.MightHTTPS()` (this func doesn't do anything if scheme is http, otherwise sets https) and use it here?

If http is already set, I think we shouldn't force convert it to https because some might be using a host where https isn't enabled.

Reply:
Yes, nice one 👍

Reply:
Added in https://github.com/ignite-hq/cli/pull/2392/commits/605fec3beb490c0ecf8df0cac58d79f566db7822